newa/__init__.py

```diff
@@ -1,6 +1,132 @@
-import click
+import io
+from enum import Enum
+from pathlib import Path
+from typing import TYPE_CHECKING, TypeVar
 
+import attrs
+import ruamel.yaml
+import ruamel.yaml.nodes
+import ruamel.yaml.representer
+from attrs import define, field
 
-@click.command()
-def main() -> None:
-    print('Newa!')
+if TYPE_CHECKING:
+    from typing_extensions import Self, TypeAlias
+
+    ErratumId: TypeAlias = str
+
+
+T = TypeVar('T')
+SerializableT = TypeVar('SerializableT', bound='Serializable')
+
+
+def yaml_parser() -> ruamel.yaml.YAML:
+    """ Create standardized YAML parser """
+
+    yaml = ruamel.yaml.YAML()
```

Comment:
Should we play safe?
```suggestion
    yaml = ruamel.yaml.YAML(typ='safe')
```

Reply:
Fixed in 4d34e7f